modules/lib/maintainers.nix

```diff
@@ -353,6 +353,12 @@
     github = "Rosuavio";
     githubId = 7164552;
   };
+  robwalt = {
+    name = "Robert Walter";
+    email = "robwalter96@gmail.com";
+    github = "RobWalt";
+    githubId = 26892280;
+  };
```

Comment:
You also need to add yourself as maintainer to the module, unless @seylerius minds.

Actually, since you are already listed in the Nixpkgs `maintainer-list.nix` file it is not necessary to also add yourself here.